<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtherMainWindow</class>
 <widget class="QMainWindow" name="AtherMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1671</width>
    <height>667</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Wareed</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Image/WareedIcon.png</normaloff>../Image/WareedIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">

QLineEdit{
background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid rgba(124, 44, 24, 219);
padding-bottom:7px;

}

</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="TeamServerTab">
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QSplitter" name="splitter">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <widget class="QTableWidget" name="TableSessions">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="tabKeyNavigation">
                <bool>true</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideRight</enum>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>20</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>200</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>SessionID</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Host Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>User Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>PID</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>OS</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Privilege</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Last Ping</string>
                </property>
               </column>
              </widget>
             </widget>
             <widget class="QTabWidget" name="TabWidgetBottom">
              <property name="currentIndex">
               <number>-1</number>
              </property>
              <property name="tabsClosable">
               <bool>true</bool>
              </property>
              <property name="movable">
               <bool>true</bool>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionActiveDomain"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1671</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHome">
    <property name="title">
     <string>Home</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menuview">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionConfigration"/>
   </widget>
   <widget class="QMenu" name="menuPayload">
    <property name="title">
     <string>Payload</string>
    </property>
    <widget class="QMenu" name="menuGenerate">
     <property name="title">
      <string>Generate</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="actionwindows"/>
     <addaction name="actionlinux"/>
    </widget>
    <addaction name="menuGenerate"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="actionabout"/>
   </widget>
   <addaction name="menuHome"/>
   <addaction name="menuview"/>
   <addaction name="menuPayload"/>
   <addaction name="menuabout"/>
  </widget>
  <action name="actionListener">
   <property name="text">
    <string>Listener</string>
   </property>
  </action>
  <action name="actionConfigration">
   <property name="text">
    <string>Configration</string>
   </property>
  </action>
  <action name="actionwindows">
   <property name="text">
    <string>windows</string>
   </property>
  </action>
  <action name="actionlinux">
   <property name="text">
    <string>linux (under dev)</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
  <action name="actionActiveDomain">
   <property name="icon">
    <iconset resource="../rc.qrc">
     <normaloff>:/icon/Image/icon/pngwing.com.png</normaloff>:/icon/Image/icon/pngwing.com.png</iconset>
   </property>
   <property name="text">
    <string>Listener</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rc.qrc"/>
 </resources>
 <connections/>
</ui>
